tests/data/tasks/profile-to-osco/input/profile-disable-0.1.58.json

```diff
@@ -0,0 +1,107 @@
+{
+  "profile": {
+    "uuid": "31182b1c-32b3-422d-a366-62b39272058c",
+    "metadata": {
+      "title": "Tailored Profile",
+      "last-modified": "2021-08-06T13:25:50.741+00:00",
+      "version": "1.0",
+      "oscal-version": "1.0.0",
+      "props": [
+        {
+          "name": "profile_mnemonic",
+          "ns": "https://my.github.io/compliance-trestle/schemas/oscal/my-cloud",
+          "value": "roks-cis-node",
+          "class": "scc_profile_mnemonic"
+        },              
+        {
+          "name": "base_profile_mnemonic",
+          "ns": "https://my.github.io/compliance-trestle/schemas/oscal/my-cloud",
+          "value": "ocp4-cis-node",
+          "class": "scc_base_profile_mnemonic"
+        },
+        {
+          "name": "profile_check_version",
+          "value": "0.1.58"
+        }            
+      ]
+    },
+    "imports": [
+      {
+        "href": "",
+        "exclude-controls": [
+          {
```

Comment:
Why no href?  Does it get filled in later?  Or is this not really used as a profile?

Reply:
Fixed.